packages/core/src/components/Button/button.module.css

```diff
@@ -17,6 +19,18 @@
 .cds-button--tertiary:not([disabled]):active {
   box-shadow: none;
 }
+.cds-button--reverseTextIcon {
+  flex-direction: row-reverse;
+}
+.cds-button__icon {
+  display: inline-flex;
+}
+.cds-button__icon--start {
+  margin-right: 10px;
```

Comment:
Could you tag in with Steven and see if there’s a token associated with this value (or if there should be one)?

Reply:
Steven was able to add a new token `--cds-button-gap`